twx

```diff
@@ -207,31 +208,45 @@ import_ext() {
     local input_params="$1"
     local url="$TWX_URL/ExtensionPackageUploader?purpose=import&validate=false"
 
-    local response=""
-    response=$(
-        curl -X POST "$http_option" -F "file=@$input_params" \
+    local http_status=$(
+        curl -X POST "$http_option" -o "response-import-ext.json" -w '%{http_code}' -F "file=@$input_params" \
             -H "appKey:$TWX_APPKEY" \
             -H "X-XSRF-TOKEN:TWX-XSRF-TOKEN-VALUE" "$url"
     )
 
-    if [[ "$response" == "Not Acceptable" ]]; then
-        echo "Fail. Not Acceptable"
-        exit_code=8
-        exit $exit_code
-    else
-        local report_message=""
-        report_message=$(echo "$response" | jq -r '.rows[0].validate.rows[0].reportMessage')
+    local http_response=$(cat response-import-ext.json)
+    rm -r "response-import-ext.json"
 
-        if [ "$report_message" == "Extension is already installed" ]; then
-            echo "Success"
-            exit_code=0
-        elif [ "$report_message" == "" ]; then
+    if [ "$http_status" -eq 200 ]; then
+        local report_status=$(echo "$http_response" | jq -r '.rows[1].install.rows[0].extensionReportStatus')
+
+        if [ "$report_status" == "2" ]; then
+            echo $(echo "$http_response" | jq -r '.rows[1].install.rows[0].reportMessage')
+        else
             echo "Success"
-            exit_code=0
+        fi
+    elif [ "$http_status" -eq 406 ]; then
+        if [[ "$http_response" == "Not Acceptable" ]]; then
+            echo "Fail. Not Acceptable"
```

Comment:
Prefix all error messages with "Error: " for consistency and update all other places in this script to do it.

Reply:
Done.